ui/src/main/resources/edu/wpi/grip/ui/templates/python/macros.vm

```diff
@@ -0,0 +1,34 @@
+#macro( input $inp $c)
```

Comment:
Use consistent formatting with whitespace around parens.
Either use `#macro(input $inp $c)` or `#macro( input $inp $c )`, but don't mix them. This goes for every use and definition of Velocity macros (`#if`, `#input`, etc.)